ci-operator/step-registry/ipi/conf/operator-publish-strategy/ipi-conf-operator-publish-strategy-commands.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="/tmp/install-config-mixed-publish.yaml.patch"
+
+cat > "${PATCH}" << EOF
+publish: Mixed
+operatorPublishingStrategy:
+  apiserver: ${APISERVER_PUBLISH_STRATEGY}
```

Comment:
If `APISERVER_PUBLISH_STRATEGY` is applied with the default value - an empty string, the real publish strategy is still "External" ?

Reply:
Or we can judge `APISERVER_PUBLISH_STRATEGY` or `INGRESS_PUBLISH_STRATEGY`, once non-empty, then set it.

Reply:
Fixed.
One of them must be set when publish:Mixed, and if another one is empty string, the default publish strategy is External.